spring-cloud-starter-dataflow-server-kubernetes/pom.xml

```diff
@@ -21,5 +21,51 @@
 			<groupId>org.springframework.cloud</groupId>
 			<artifactId>spring-cloud-dataflow-server-core</artifactId>
 		</dependency>
+		<dependency>
```

Comment:
Also, I think it would be more correct to use `project.name` rather than `project.description` (and have `project.name` not be the lower case thing that `artifactId` already is